src/app/modules/project/overview/overview.component.ts

```diff
@@ -149,26 +148,27 @@ export class OverviewComponent implements OnInit {
     private paginationService: PaginationService,
     private internalSearchService: InternalSearchService,
     private formBuilder: FormBuilder,
-    private activatedRoute: ActivatedRoute) {
+    private activatedRoute: ActivatedRoute
+  ) {
     this.searchControl = new FormControl('');
 
     this.categoryForm = this.formBuilder.group({
       code: false,
       video: false,
       research_paper: false,
-      survey_results: false
+      survey_results: false,
     });
 
     this.tagsForm = this.formBuilder.group({
       learning: false,
       research: false,
       mobile: false,
-      ux: false
+      ux: false,
     });
 
     this.sortForm = this.formBuilder.group({
       type: this.sortTypeSelectOptions[0].value,
-      direction: this.sortDirectionSelectOptions[0].value
+      direction: this.sortDirectionSelectOptions[0].value,
```

Comment:
Since the default sort option is name, I would aspect the default direction to be asc. So my suggestion is to change the default sorting to created. Beside that I think that when the sort type gets change to Name, the default direction should be change with the code to asc

Reply:
I would personally use updated - descending. It means that the projects which are updated or created last are on top.
This is also the default in the backend.

Reply:
i agree with stijn that when you go to the overview page the newest projects should be on top.
github & gitlab also do this.

Reply:
I've made changes so that it's now updated - descending. Thanks for the discussion everyone.